opennms-model/src/main/java/org/opennms/netmgt/model/discovery/IPAddrRange.java

```diff
@@ -59,6 +59,8 @@
     private static final long serialVersionUID = -106414771861377679L;
 
     private static final Logger LOG = LoggerFactory.getLogger(IPAddrRange.class);
+    
+    public static final double FUDGE_FACTOR = 1.5;
```

Comment:
Make this constant a BigDecimal instead of a double. Move the constant into DiscoveryJob.